lib/config.coffee

```diff
@@ -74,7 +74,7 @@ exports.generate = (options, params = {}) ->
 		appUpdatePollInterval: params.appUpdatePollInterval or 60000
 
 		listenPort: 48484
-		vpnPort: options.vpnPort or 1723
+		vpnPort: _.parseInt(options.vpnPort) or 1723
```

Comment:
why would a random string translate to 1723? I propose to throw an error if it cannot be parsed as a number